examples/connext_dds/lbediscovery/README.md

```diff
@@ -0,0 +1,67 @@
+# Example Code: LBED and XML-Based Application Creation
+
+## Concept
+
+The *RTI Limited Bandwidth Endpoint Discovery (LBED) Plugin* reduces discovery
+time and network traffic by locally defining information about the endpoints
+that need to be discovered in an XML file. The default dynamic discovery process,
+on the other hand, sends the information about the endpoints over the network.
+Therefore, LBED requires all the endpoints to be known ahead of time and each
+must be declared in an XML file.
+
+One of the requirements for using LBED in an application is that every endpoint
+must define a value for the RTPS object ID. The RTPS object ID is an integer
+that uniquely identifies an endpoint of a specific kind (DataWriter or DataReader)
+within a DomainParticipant. Normally, the RTPS object ID is automatically assigned
+by Connext when an endpoint is created, and it is exchanged during the Simple
+Endpoint Discovery phase. However, the LBED Plugin replaces Simple Endpoint
+Discovery, so this auto-assigned RTPS object ID will not be propagated to other
+endpoints. In order for remote endpoints to be discovered, LBED needs to know
+the RTPS object ID of a remote participant's entities before discovery initiates.
+
+If your application uses XML-Based Application Creation, you do not have to
```

Comment:
If your application uses XML-Based Application Creation, however, you do not have to